docu/SICKAppSpaceCodingStarterKit_Documentation.md

```diff
@@ -667,6 +713,451 @@ Script.serveFunction("CSK_ModuleName.setSelectedInstance", setSelectedInstance)
+      CSK_UserManagement = require 'API.CSK_UserManagement'
+    elseif appList[i] == 'CSK_Module_FlowConfig' then
+      CSK_FlowConfig = require 'API.CSK_FlowConfig'
+    end
+  end
+...
+```
+
+- Within the main file, require the main FlowConfig file (regarding the FlowConfig files, see chapter [FlowConfig Files](#flowconfig-files)) and hand over the handle of the instances.
+
+```
+local setInstanceHandle = require('MainFolder/ModuleName/FlowConfig/ModuleName_FlowConfig')
+...
+setInstanceHandle(multiDataLogger_Instances)
+```  
+
+- Module should provide a function 'setFlowConfigPriority' to set the status if the FlowConfig should have priority (see chapter [FlowConfig Parameter Logic](#flowconfig-parameter-logic) ) and an event named 'OnNewStatusFlowConfigPriority' to provide this status.
+
+- Module should provide a function 'getStatusModuleActive' to inform the CSK_Module_FlowConfig if the features of the module are available on the device the module is running on.
+
+- Module should provide a function 'clearFlowConfigRelevantConfiguration' to clear all FlowConfig relevant configuration if the FlowConfig priority is active for the module (see chapter [FlowConfig Parameter Logic](#flowconfig-parameter-logic) ).
+
+### FlowConfig Parameter Logic
+
+Within the FlowConfig it is possible to link the features of CSK modules and to set some related parameters for these modules.
+
+Because of this, it is possible that the setup of the FlowConfig differs from the setup done within the specific module and could lead to unexpected behaviour.
+
+To prevent this, there is the "FlowConfig priority". The idea of this priority is to clear all configurations of the module which can be done via FlowConfig as well. That means, if the priority is active, these configurations are only controlled via FlowConfig.
+
+If the user is aware of this possible vulnerability, it is still possible to deactivate that priority and to combine module specific configuration with the FlowConfig setup. It is just important that these configurations are in harmony with each other.
+
+In general the logic of loading the parameters via the CSK_Module_PersistentData (see [Persistent Data](#persistent-data) ) is like this:
+
+1) First, each CSK module will load its saved configuration if its 'Load on reboot' status is active (see modules UI)
+
+2) After that, the FlowConfig will load its configuration
+
+3) If the 'FlowConfig priority' status of a module is active, all FlowConfig relevant configurations of these modules will be cleared (the module developer needs to take care of this, see 'clearFlowConfigRelevantConfiguration' function).
+
+4) The configuration stored within the FlowConfig will be set to the related CSK modules
+
+### FlowConfig Files
+
+If the module shoud support the FlowConfig feature, there is even some more additional content to add.
+
+**INFO: Before you try to extend a module with this feature, you should be sure to understand all the other CSK basics!**
```

Comment:
Like mentioned above:
OK, but alternatively to me "other" sounds strange, Optional suggestion: 
"understand all the basics of CSK"

Reply:
Done